
Xvnc Free Edition 4.1.1 - built Feb 25 2015 23:02:21
Copyright (C) 2002-2005 RealVNC Ltd.
See http://www.realvnc.com for information on VNC.
Underlying X server release 40300000, The XFree86 Project, Inc


Fri Jan 10 10:15:21 2020
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on port 5914
 vncext:      created VNC server for screen 0
error opening security policy file /etc/X11/xserver/SecurityPolicy
Could not init font path element /usr/X11R6/lib/X11/fonts/Type1/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/Speedo/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/misc/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/75dpi/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/100dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/75dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/100dpi/, removing from list!
xrdb: No such file or directory
xrdb: can't open file '/home/is28/fedin28/.Xresources'
/usr/bin/startxfce4: X server already running on display :14
gpg-agent[32155]: WARNING: "--write-env-file" is an obsolete option - it has no effect
gpg-agent: a gpg-agent is already running - not starting a new one

(xfce4-session:32145): xfce4-session-WARNING **: 10:15:24.529: gpg-agent returned no PID in the variables

(xfce4-session:32145): xfce4-session-WARNING **: 10:15:24.537: xfsm_manager_load_session: Something wrong with /home/is28/fedin28/.cache/sessions/xfce4-session-virt-1:14, Does it exist? Permissions issue?

(xfwm4:32159): xfwm4-WARNING **: 10:15:24.577: The display does not support the XRender extension.

(xfwm4:32159): xfwm4-WARNING **: 10:15:24.577: The display does not support the XComposite extension.

(xfwm4:32159): xfwm4-WARNING **: 10:15:24.577: The display does not support the XDamage extension.

(xfwm4:32159): xfwm4-WARNING **: 10:15:24.577: The display does not support the XFixes extension.

(xfwm4:32159): xfwm4-WARNING **: 10:15:24.577: Compositing manager disabled.

** (light-locker:32169): ERROR **: 10:15:24.671: Environment variable XDG_SESSION_PATH not set. Is LightDM running?
(uint32 1,)

(polkit-gnome-authentication-agent-1:32171): GLib-CRITICAL **: 10:15:24.727: g_variant_new_string: assertion 'string != NULL' failed

(polkit-gnome-authentication-agent-1:32171): polkit-gnome-1-WARNING **: 10:15:24.729: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
xfwm4-Message: 10:15:24.755: Unsupported keyboard modifier '<Super>Tab'

(xfsettingsd:32168): xfsettingsd-CRITICAL **: 10:15:24.805: RANDR extension is too old, version 1.1. Display settings won't be applied.
Xlib:  extension "XInputExtension" missing on display ":14.0".

(xfsettingsd:32168): xfsettingsd-CRITICAL **: 10:15:24.819: XI is not present.

(xfsettingsd:32168): xfsettingsd-CRITICAL **: 10:15:24.819: Failed to initialize the Xkb extension.

(xfsettingsd:32168): xfsettingsd-CRITICAL **: 10:15:24.820: Failed to initialize the Accessibility extension.
xfwm4-Message: 10:15:24.915: Unsupported keyboard modifier '<Super>Tab'

(nm-applet:32265): Gtk-WARNING **: 10:15:25.099: Can't set a parent on widget which has a parent

(nm-applet:32265): Gdk-CRITICAL **: 10:15:25.276: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
/usr/lib/python3/dist-packages/blueman/plugins/applet/AppIndicator.py:8: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3 as girAppIndicator
ERROR:dbus.proxies:Introspect error on org.bluez:/org/bluez: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.15405" (uid=1285 pid=32232 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.15405" (uid=1285 pid=32232 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.15405" (uid=1285 pid=32232 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)

(wrapper-1.0:32324): Gdk-WARNING **: 10:15:30.787: /build/gtk+2.0-AoeliP/gtk+2.0-2.24.32/gdk/x11/gdkdrawable-x11.c:952 drawable is not a pixmap or window

Fri Jan 10 10:15:41 2020
 Connections: accepted: 0.0.0.0::51249
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)

Fri Jan 10 10:15:46 2020
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

(Thunar:32165): Gdk-WARNING **: 10:15:53.548: gdk_window_set_icon_list: icons too large

(gedit:32535): Gtk-WARNING **: 10:17:16.501: Calling org.xfce.Session.Manager.Inhibit failed: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method "Inhibit" with signature "susu" on interface "org.xfce.Session.Manager" doesn't exist


Fri Jan 10 10:25:09 2020
 Connections: closed: 0.0.0.0::51249 (Clean disconnection)
 SMsgWriter:  framebuffer updates 554
 SMsgWriter:    hextile rects 1233, bytes 6743130
 SMsgWriter:    raw bytes equivalent 90202332, compression ratio 13.376923
/usr/lib/ubuntu-release-upgrader/check-new-release-gtk:30: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
WARNING:root:timeout reached, exiting
/usr/share/system-config-printer/applet.py:45: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
system-config-printer-applet: failed to start NewPrinterNotification service
system-config-printer-applet: failed to start PrinterDriversInstaller service: org.freedesktop.DBus.Error.AccessDenied: Connection ":1.15406" is not allowed to own the service "com.redhat.PrinterDriversInstaller" due to security policies in the configuration file
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 10 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 14 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 16 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 22 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 26 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 27 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 31 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 33 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 35 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 36 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 40 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 42 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 48 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 52 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 54 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 58 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 60 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 61 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 65 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 66 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 70 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 72 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 73 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 77 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 79 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 81 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 87 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 91 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 93 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 94 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 98 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 100 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 101 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 105 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 107 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 108 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 112 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 114 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 115 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 119 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 121 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 122 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 126 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 127 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 133 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 134 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 138 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 140 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 142 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 143 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 147 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 149 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/lib/ubuntu-release-upgrader/check-new-release-gtk:30: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
WARNING:root:timeout reached, exiting
xfwm4: Fatal IO error 4 (Interrupted system call) on X server :14.0.
xfdesktop: Fatal IO error 11 (Resource temporarily unavailable) on X server :14.0.
